models/conditional_gans.py

```diff
@@ -116,9 +116,8 @@ def generate_conditionals(self, embedding):
         """
         # NOTE embedding is dim (batch, 10, 1024) where 10 is different samples for same image. Options are to either
         # flatten all features or average across embeddings
-        x = embedding[:, 0, :] # rather just select one of the embeddings
-        mean = self.lrelu1(self.dense_mean(x))
-        sigma = self.lrelu2(self.dense_sigma(x))
+        mean = self.lrelu1(self.dense_mean(embedding))
+        sigma = self.lrelu2(self.dense_sigma(embedding))
```

Comment:
Which dimension do `dense_mean` and `dense_sigma` operate over?

Reply:
The embedding dimension (1024)